packages/caravan-health/src/waste.test.ts

```diff
@@ -0,0 +1,82 @@
+import { feesToAmountRatio, relativeFeesScore, wasteMetric } from "./waste";
+import { Transaction, FeeRatePercentile } from "@caravan/clients";
+
+describe("Waste metrics that accounts for nature of fees spending for a wallet", () => {
+  it("Relative fees score for transaction with respect to international fiat payment charges", () => {
+    const transactions: Transaction[] = [
+      {
+        vin: [],
+        vout: [],
+        txid: "tx1",
+        size: 0,
+        weight: 1,
+        fee: 1,
+        isSend: true,
+        amount: 0,
+        blocktime: 1234,
+      },
+    ];
+    const feeRatePercentileHistory: FeeRatePercentile[] = [
+      {
+        avgHeight: 0,
+        timestamp: 1234,
+        avgFee_0: 0,
+        avgFee_10: 0,
+        avgFee_25: 0.5,
+        avgFee_50: 1,
+        avgFee_75: 0,
+        avgFee_90: 0,
+        avgFee_100: 0,
+      },
+    ];
+    const score: number = +relativeFeesScore(
+      transactions,
+      feeRatePercentileHistory,
+    ).toFixed(3);
```

Comment:
fwiw, I tested and found that the tests (both in this file) still pass like this:

```suggestion
    const score: number = relativeFeesScore(
      transactions,
      feeRatePercentileHistory,
    );
```

Reply:
So this works because for this test case the float value is not that decimated. Try with the UTXO Spread Factor. It has variance calculations so that raises exceptions even when comparing upto 5 decimal places as `number`. 

For now what I will do is remove the precisions wherever we can.

Reply:
or maybe i can calculate expected score from raw formula.